### DISCUSSION TITLE
CronJob Marketplace

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#500] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Hi! We recently had to create a cronjob and we noticed that it is a more complicated process compared to creating a new microservice. I understand that cronjobs are not as common as microservices, but it would still be nice to have access to a small marketplace with templates for cronjobs. The templates would basically be a stripped down version of the microservices templates, with fewer dependencies and a `Hello world!` log instead of the usual http server bootstrapping code. I presume that almost all of the other components would remain the same (Dockerfile, pipeline, package manager, tests, etc.)

**Comment by fredmaggiowski** [comment]
Hi @GioAc96 thanks for the feedback, there is already a story in the internal roadmap for this, it just didn't made it to our plannings yet 😞

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
